action_server/src/sema4ai/action_server/_new_project.py

```diff
@@ -83,3 +87,51 @@ def create_new_project(directory: str = ".", template_name: str = "") -> int:
         )
 
         return 1
+
+
+def list_templates(print_json: bool = False) -> int:
+    """Lists availabe templates.
+
+    Args:
+        print_json: If true, the output will be formatted as JSON.
+    """
+    from sema4ai.action_server.vendored_deps.termcolors import bold_red
+
+    from ._new_project_helpers import (
+        _ensure_latest_templates,
+        _get_local_templates_metadata,
+        _print_templates_list,
+    )
+
+    try:
+        _ensure_latest_templates()
+        metadata = _get_local_templates_metadata()
+
+        if not metadata or len(metadata.templates) == 0:
```

Comment:
Humm, not sure it should raise an error here. For this function returning `[]` or printing `No templates available` seems fine.

Reply:
I was also on the fence - decided however, that because no templates would make `new` command itself raise an error, decided to do the same here. Let me know what you think !

Reply:
I think it makes sense for new because it cannot create a project without a template, but for listing an empty list is something valid.

Reply:
Fair - I will update it!